src/jetson/main.py

```diff
@@ -297,28 +304,28 @@ def encryptFrame(img:np.ndarray,
     g.eval()
     class_names = ['Glasses', 'Goggles', 'Neither']
 
-    cap = VideoCapturer() #Instantiate Video Capturer object
+    cap = VideoCapturer() 
     detector = FaceDetector(detector=args.detector, cuda=args.cuda and torch.cuda.is_available(), set_default_dev=True) #Instantiate Face Detector object
-    cl = Classifier(g) #Instantiate Classifier object
+    cl = Classifier(g)
+    enc = Encryptor() 
 
     while True:
```

Comment:
this needs a comment. Never write `while True` without an explicit comment of its need. 

on a side note, `while True` is bad. it hard to read and all it does is raise alarm flags for me as a developer. I'd much rather see something like 

```python
run_face_detection: bool = True
while run_face_detection:
    # execute code here
    if stop_condition:
        run_face_detection = False
```

I know its more code but its makes it more clear what is happening here. readability is usually better than conciseness when writing code